web-mode/vue

```diff
@@ -0,0 +1,18 @@
+# -*- mode: snippet -*-
+# name: vue
+# uuid: vue
+# key: vue
+# condition: t
+# --
+<script>
+  export default {
+    name: '${0:`(f-base buffer-file-name)`}'
```

Comment:
Replace `f-base` with `file-name-base`. To avoid needlessly depending on the `f` library (and imposing that dependency on users).

Reply:
I'm not very familiar with elisp, so thanks for clarifying this to me.